css-view-transitions-2/Overview.bs

```diff
@@ -939,15 +992,20 @@ It has the following [=struct/items=]:
 
 ### Captured elements extension ### {#capture-classes-data-structure}
 The [=captured element=] struct should contain these fields, in addition to the existing ones:
-	<dl>
-		: <dfn for="captured element">class list</dfn>
+	<dl dfn-for="captured element">
+		: <dfn>class list</dfn>
 		:: a [=/list=] of strings, initially empty.
 
-		: <dfn for="captured element">containing group name</dfn>
+		: <dfn>containing group name</dfn>
 		:: Null or a string, initially null.
 
-		: <dfn for="captured element">transform from snapshot containing block</dfn>
-		:: A [=matrix=], initially the identity matrix.
+		: <dfn>old layered-capture style</dfn>
+		:: A string.
+
+		: <dfn>transform from snapshot containing block</dfn>
+		: <dfn>old nested group transform
```

Comment:
why do we need new transforms here...? we dealt with this for nesting, layered capture shouldn't need to track new transforms.